.github/workflows/tests.yaml

```diff
@@ -25,6 +30,11 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         pyv: ["3.11"]
+        vllm_version:
+          # - "pypi" # skip the pypi version as it will not work with CPU
```

Comment:
We cannot run the pypi version because it will try to use cuda even if `--device=cpu` is passed. This is due to how `vllm` is built and internally checks for cpu support (i.e by checking if `cpu` is in the vllm version.

Reply:
Luckily, by providing `VLLM_TARGET_DEVICE` and not building the avx512 version, we can quickly build the wheel from source .